e2e/paths/enterOrdersAndDirections_test.js

```diff
@@ -0,0 +1,23 @@
+const config = require('../config.js');
+
+Feature('Enter order and details');
+
+Before((I, caseViewPage) => {
+  I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventSummary);
+  caseViewPage.goToNewActions(config.applicationActions.selectOrders);
+});
+
+Scenario('Select care order', 
+(I, ordersNeededPage) => {
```

Comment:
we havent broke line for our Scenario, so try keep it on 1 line
```suggestion
Scenario('Select care order', (I, ordersNeededPage) => {
```